baram/ec2_manager.py

```diff
@@ -13,6 +13,50 @@ def list_security_groups(self):
         '''
         return self.cli.describe_security_groups()['SecurityGroups']
 
+    def list_instances(self):
+        """
+        Describes all instances currently using
+
+        :return:
+        """
+        instances = self.cli.describe_instances()['Reservations']
+        result = [x['Instances'][0] for x in instances]
+        return result
+
+    def delete_redundant_key_pairs(self):
+        """
+        Delete redundant key pairs (i.e. not related to any instances)
+        """
+        key_pairs_redundant = self.list_redundant_key_pairs()
+        number_of_key_pairs_redundant = len(key_pairs_redundant)
+        if number_of_key_pairs_redundant == 0:
+            print("There's no redundant key pair")
+        else:
+            for key_pair in key_pairs_redundant:
+                self.cli.delete_key_pair(KeyName=key_pair)
+            print(f"{number_of_key_pairs_redundant} redundant key pairs has deleted")
```

Comment:
please don't use print in production code. Instead, use log manager.

Reply:
Yes. It'll be reflected to [this](https://github.com/lks21c/baram/pull/31) shortly.